src/components/AddRepoForm/AddRepoForm.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux'
+import { getRepository } from '../../actions'
+import { Form, InputGroup, FormControl, Button } from 'react-bootstrap';
+import './AddRepoForm.scss'
+
+const AddRepoForm = () => {
+  const [repoName, setRepoName] = useState('')
+  const error = useSelector(state => state.repositories.error)
+  const dispatch = useDispatch()
+
+  const handleSubmit = (e) => {
+    e.preventDefault()
+    dispatch(getRepository(repoName))
+    setRepoName('')
+  }
+
+  const handleChange = (e) => {
+    setRepoName(e.target.value)
+  }
+
+  return(
+    <Form
+      className="add-repo-form"
+      onSubmit={handleSubmit}>
+      <InputGroup className="mb-3">
+        <FormControl
```

Comment:
Usually we use library which handles forms https://github.com/formium/formik, but it's ok for this small project